workflow/deterministic_wrappers.go

```diff
@@ -106,6 +106,18 @@ func Now(ctx Context) time.Time {
 // is canceled, the returned Future become ready, and Future.Get() will return *CanceledError.
```

Comment:
we already mention the cancelled case here.

Reply:
i believe by documenting some common pitfalls, developers will have less chance wonder what happened.